notebooks/00_core.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<p>just an aesthetic remark: if you'd delete the data before committing the notebook we'd see the default message.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>good idea! will implement it</p>